esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -217,6 +219,62 @@ bool CLoggingManager::updateLog(IEspContext* espContext, IEspUpdateLogRequestWra
     return bRet;
 }
 
+static bool checkEnabledLogEntry(IPropertyTree *scriptValues, const char *profile, const char *group, unsigned &groupEntryCount, const char *logtype, unsigned &typeEntryCount)
+{
+    if ((!group || *group) && (!logtype || !*logtype)) //get this special case out of the way
+        return false;
+    //we only care about groups if a profile is specified, if so does it match?
+    if (profile && *profile && group && *group)
+    {
+        groupEntryCount++;
+        if (!strieq(profile, group))
+            return false; //don't need to fall through and check types this group doesn't match
+    }
+    //if an entry has an enabled group but no type that entry is enabled
+    if (!logtype || !*logtype)
+        return true;
+    typeEntryCount++;
+    //if our type isn't disabled then this entry is enabled
+    VStringBuffer xpath("@disable-log-type-%s", logtype);
+    return (!scriptValues->getPropBool(xpath, false));
+}
+
+bool checkSkipThreadQueue(IPropertyTree *scriptValues, IUpdateLogThread &logthread)
```

Comment:
It may be very helpful in this function (and checkEnabledLogEntry) to print a message (when at high log levels) when a log agent is skipped and for what reason.